.travis.yml

```diff
@@ -6,10 +6,15 @@ sudo: required
 before_install:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
+install:
+  - npm install -g codecov
+  - npm install
 before_script:
   - npm run download-sync-client
 script:
   - npm run testsuite
+after_script:
+  - npm run unittest-cov && codecov
```

Comment:
Yeah this is the last cycle in build lifecycle, so this should be executed when scripts are finished.